README.md

```diff
@@ -70,6 +70,8 @@ module.exports = {
       plugins: ['ember'],
       extends: [
         'eslint:recommended',
+        'plugin:ember/recommended',
+        'plugin:ember/gjs-recommended',
```

Comment:
both gjs and gts for gts files? 🤷 why not the one that matches the extension?

Reply:
Can  also be done. Just means we have to add rules to 2 configs